tensorboard/webapp/runs/views/runs_table/runs_table_container.ts

```diff
@@ -694,11 +695,12 @@ export class RunsTableContainer implements OnInit, OnDestroy {
     );
   }
 
-  onRunSelectionDblClick(item: RunTableItem) {
-    // Note that a user's double click will trigger both 'change' and 'dblclick'
-    // events so onRunSelectionToggle() will also be called and we will fire
-    // two somewhat conflicting actions: runSelectionToggled and
-    // singleRunSelected. This is ok as long as singleRunSelected is fired last.
+  onRunSelectionDblClick(runId: string) {
+    // Note that a user's double click in the Legacy RunsTableComponent will
```

Comment:
Not sure if it's worth doing but it would be sort of nice to fix the legacy runs_table_component to use a similar implementation to the runs_data_table,  just so you could delete this entire comment. (If you decide to do it, it could be left to a different PR).

Reply:
I thought that would be a good follow up as well.